packages/engine/src/avatar/components/LoopAnimationComponent.ts

```diff
@@ -113,7 +113,7 @@ export const LoopAnimationComponent = defineComponent({
      * A model is required for LoopAnimationComponent.
      */
     useEffect(() => {
-      if (!modelComponent?.scene?.value) return
+      if (!modelComponent?.scene?.value || !modelComponent.asset.value) return
```

Comment:
if we are checking `scene` and `asset` properties, why aren't we including both as useEffect dependencies?

Reply:
Forgot to remove the extra check there. Checking both is redundant.